src/amo/pages/Addon/index.js

```diff
@@ -535,6 +536,7 @@ export class AddonBase extends React.Component {
                       hasAddonManager={hasAddonManager}
                       install={install}
                       installTheme={installTheme}
+                      setCurrentStatus={setCurrentStatus}
```

Comment:
I don't see this being used in the AMInstallButton - is that something you're adding in a later PR?

Reply:
nope, but it needs to be passed.